e2e/app/component/runtime-panel.ts

```diff
@@ -0,0 +1,192 @@
+import Container from 'app/container';
+import {LinkText} from 'app/text-labels';
+import Button from 'app/element/button';
+import {Page} from 'puppeteer';
+import {waitForAttributeEquality, waitWhileLoading} from 'utils/waits-utils';
+import PrimereactInputNumber from 'app/element/primereact-input-number';
+import SelectMenu from "./select-menu";
+import {savePageToFile, takeScreenshot} from "../../utils/save-file-utils";
+import BaseElement from "../element/base-element";
+
+const defaultXpath = '//*[@id="runtime-panel"]';
+const startStopIconXpath = '//*[@data-test-id="runtime-status-icon"]';
+
+export enum StartStopIconState {
+  Error = 'error',
+  None = 'none',
+  Running = 'running',
+  Starting = 'starting',
+  Stopped = 'stopped',
+  Stopping = 'stopping'
+}
+
+export enum ComputeType {
+  Standard = 'Standard VM',
+  Dataproc = 'Dataproc Cluster'
+}
+
+export enum RuntimePreset {
+  GeneralAnalysis = 'General Analysis',
+  HailGenomicsAnalysis = 'Hail Genomics Analysis'
+}
+
+export default class RuntimePanel extends Container {
```

Comment:
TOL I think we need a sidebar base class which runtime panel and help sidebar can extends from.